test/e2e/logs_include_namespaces_test.go

```diff
@@ -0,0 +1,104 @@
+//go:build e2e
+
+package e2e
+
+import (
+	"net/http"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	kitk8s "github.com/kyma-project/telemetry-manager/test/testkit/k8s"
+	. "github.com/kyma-project/telemetry-manager/test/testkit/matchers/log"
+	"github.com/kyma-project/telemetry-manager/test/testkit/mocks/backend"
+	"github.com/kyma-project/telemetry-manager/test/testkit/mocks/loggen"
+	"github.com/kyma-project/telemetry-manager/test/testkit/periodic"
+	"github.com/kyma-project/telemetry-manager/test/testkit/verifiers"
+)
+
+var _ = Describe("Logs Include Namespaces", Label("logs"), Ordered, func() {
+	const (
+		mockNs          = "log-include-namespaces-mocks"
+		systemNamespace = "kyma-system"
+		mockBackendName = "log-receiver-include-namespaces"
+		logProducerName = "log-producer-include-namespaces"
+		pipelineName    = "pipeline-include-namespaces-test"
+	)
+	var telemetryExportURL string
+
+	makeResources := func() []client.Object {
+		var objs []client.Object
+		objs = append(objs, kitk8s.NewNamespace(mockNs).K8sObject())
+
+		mockBackend := backend.New(mockBackendName, mockNs, backend.SignalTypeLogs)
+		mockLogProducer := loggen.New(logProducerName, mockNs)
+		objs = append(objs, mockBackend.K8sObjects()...)
+		objs = append(objs, mockLogProducer.K8sObject(kitk8s.WithLabel("app", "logging-include-namespaces")))
+		telemetryExportURL = mockBackend.TelemetryExportURL(proxyClient)
+
+		logPipeline := kitk8s.NewLogPipeline(pipelineName).
+			WithSecretKeyRef(mockBackend.HostSecretRef()).
+			WithHTTPOutput().
+			WithIncludeNamespaces([]string{"kyma-system", mockNs}).
+			WithExcludeContainers([]string{logProducerName})
```

Comment:
Why are we excluding the log producer container? We do not assert it anywhere or am I missing something?

Reply:
Good catch, that was copy/pasted from another test by accident.